spec/helpers/summary_helper_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe SummaryHelper do
+  describe 'start_date' do
+    let(:prison_dates) do
+      [
+          { offender_no: "A1120GH", days_count: 199 },
+          { offender_no: "G9765JZ", days_count: 50 }
+      ]
+    end
+
+    it "formats an offender's prison start date if offender exists" do
+      expect(start_date(prison_dates, "A1120GH")).to eq(199)
+    end
+
+    it "formats an offender's prison start date if offender does not exist" do
+      expect(start_date(prison_dates, "Z1234WQ")).to eq('-')
```

Comment:
Presumably we wouldn't ever actually see this offender in the list, so this is the only way to ever see '-'?  

I think returning a string or an integer might be confusing, so perhaps we can just choose a sentinel value (like 0) as it'll never actually be shown.